sources/OVPSDK.swift

```diff
@@ -406,23 +406,27 @@ public struct OVPSDK {
         }
         
         func setupVRMWithNewCore() {
+            let createRequest: (URL) -> (URLRequest) = {
+                .init(url: $0, cachePolicy: .useProtocolCachePolicy, timeoutInterval: hardTimeout)
```

Comment:
.useProtocolCachePolicy - is a default value?

Reply:
dropped